frontend/app/views/fragments/form/cardDetail.scala.html

```diff
@@ -3,17 +3,22 @@
 @import com.gu.memsub.Current
 @import com.gu.memsub.subsv2.CatalogPlan.PaidMember
 @import com.gu.memsub.subsv2.MonthYearPlans
-@(plans: MonthYearPlans[PaidMember], cardOpt: Option[com.gu.stripe.Stripe.Card] = None)
+@(
+    plans: MonthYearPlans[PaidMember],
+    cardOpt: Option[com.gu.stripe.Stripe.Card] = None,
+    aDistribution : Boolean = true
+)
 
 @import views.support.Pricing._
 @import views.support.{Asset, Dates, selectDates}
 
-<fieldset class="fieldset">
+<fieldset class="fieldset" xmlns="http://www.w3.org/1999/html">
```

Comment:
@SiAdcock Neither do I. I don't remember adding that line, maybe the IDE added them. Deleted in the following commit.